src/commands/upload_file.test.ts

```diff
@@ -0,0 +1,54 @@
+import { randomUUID } from 'crypto';
+import { expect } from 'chai';
+import { ArFSResult, ArweaveAddress, EntityID, TransactionID, Winston } from 'ardrive-core-js';
+
+import { formatResults } from './upload_file';
+
+const createResultMock = (): ArFSResult => ({
+	created: [
+		{
+			type: 'file',
+			metadataTxId: (randomUUID() as unknown) as TransactionID,
```

Comment:
@fedellen @arielmelendez I was not sure if it's ok to do that or if it's expected that I create a real TransactionID. As it's a test and the results doesn't matter (they just need to be different) I went with the easy way :D

Reply:
Should be fine for this as the values of these fields aren't relevant to the test. But it could confuse some dev into later thinking that transaction IDs are UUIDs, which they are not. Maybe add a comment specifying that these TxIDs and the ArweaveAddress are invalid, but their validity is not important to the test.

 Thanks for adding coverage for this method!

Reply:
Sure :D I was thinking on using something like https://github.com/fent/randexp.js to generate strings from the regex that ardrive-core-js have, but I thought that it's not worth it :D 
I'll add those comments.

Reply:
So I have a few issues with this approach:
![image](https://user-images.githubusercontent.com/7692385/144698196-c84dae96-7079-4177-a9c7-46e397722d67.png)

1) Prettier and ESLint seem to be fighting over how this should be formatted. One wants parens around the `unknown` cast, and the other does not, resulting in a Problem in the IDE.
2) The choice to do things this way disguised a small bug on line 45:
```
		const firstFeeFromSecondResult = Object.keys(firstResult.fees)[0];
```
should be:
```
		const firstFeeFromSecondResult = Object.keys(secondResult.fees)[0];
```
But the test passes either way, and for at least two reasons that I can think of:
a) since we're not using literals, it's hard to see where something could go wrong
b) We're writing programmatic code that changes on each test run and hoping that the code we write to handle the values generated by that code are both correct AND actually test what needs to be tested.

I think it would be much simpler and straightforward to use some POJO literals with easy-to-see-and-comprehend values to validate that the function works as expected.

For what it's worth, there's a single stub exported from ardrive-core-js. But I don't think it's flexible enough or worth the effort of making flexible over, say, just creating the objects in place, to use here where we need multiple distinct values to be sure we're getting back the right values.
```
export const stubTransactionID = TxID('0000000000000000000000000000000000000000000');
```

Reply:
Take a look at the suggestion I posted as an alternative above. Thanks!